### ISSUE TITLE
After upgrade of b2handle from version 1.0x to 1.1 it stops functioning on Centos 6.8

### ISSUE THREAD

**Issue by cookie33** [EUDAT-B2SAFE/B2HANDLE#117]
HI,

After building and installing the b2handle library on Centos 6.8 with Python 2.6 it stops functioning.
Building was done as follows:
```
python setup.py bdist_rpm
```
This so we can install an rpm and see easily what is installed.

The first error is:
```
./testB2SafeCmd.py -test epic2
Test Epicclient2 Script
Test that create handle returns expected response and adds ... Traceback (most recent call last):
  File "../../cmd/epicclient2.py", line 15, in <module>
    from b2handle.handleclient import EUDATHandleClient
  File "/usr/lib/python2.6/site-packages/b2handle/handleclient.py", line 9, in <module>
    from past.builtins import range
```
The second was about the `future.standard_library` in `utilhandle.py`.

I fixed these with a try except and pass if ImportError but now we have an other error:
```
  File "/usr/lib/python2.6/site-packages/b2handle/utilhandle.py", line 14, in <module>
    from urllib.parse import quote
ImportError: No module named parse
```
The version of url lib on my system is:
```
rpm -qa | grep urllib
python-urllib3-1.10.2-1.el6.noarch
```

So for the moment I am reverting back to version 1.0.x

**Comment by cookie33** [comment]
installed python2-future
```
yum install python2-future
```
Than my tests perform better, but still one error:
```
./testB2SafeCmd.py -test epic2
Test Epicclient2 Script
Test that create handle returns expected response and adds ... Trueok
Test that create handle with checksum returns expected ... Trueok
Test that create handle with extra key returns expected ... Trueok
Test that create handle with extra key EUDAT/ROR and value 'PID' returns expected ... Trueok
Test that create handle with extra key EUDAT/ROR and value 'pid' returns expected ... Trueok
Test that create handle with extra key EUDAT/ROR returns expected ... Trueok
Test that create handle with extra location returns expected ... Trueok
Test that delete existing handle returns True. ... ok
Test that delete key from handle returns True. ... Trueok
Test that delete non existing handle returns False. ... FAIL
Test that delete non existing key from handle returns False. ... ok
Test that get value by non existing key returns None. ... Trueok
Test that get value by key returns stored handle value. ... Trueok
Test that update handle with same location value returns None. ... Trueok
Test that modify handle value returns True and updates ... Trueok
Test that modify existing handle with new key-value pair ... Trueok
Test that modify existing handle key with empty value ... Trueok
Test that modify value of non existing handle returns False. ... ok
Test that retrieve non existing handle returns None. ... ok
Test that search by key-value returns matching handle. ... Trueok
Test that search handle by non existing key-value returns 'empty'. ... ok
Test that update handle with new location returns None and ... Trueok
Test that update non existing handle with location returns False. ... ok

======================================================================
FAIL: Test that delete non existing handle returns False.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robertv/git/B2SAFE-core/scripts/tests/testB2SafeCmd/epic2intgtest.py", line 326, in test_delete_non_existing_handle
    'delete non existing handle should return False')
AssertionError: 'True' != 'False' : delete non existing handle should return False

----------------------------------------------------------------------
Ran 23 tests in 30.918s

FAILED (failures=1)
```

**Comment by cookie33** [comment]
This is the same as the issue: deleting a non-existent handle does not raise an exception #109

**Comment by TobiasWeigel** [comment]
OK, I looked at the codebase - this apparently fell between the cracks in commit 19f81e.

**Comment by SofianeB** [comment]
The  fix #109 by Tobias is not in version 1.1.0.
